aptos-move/e2e-move-tests/src/tests/aggregator_v2.rs

```diff
@@ -231,8 +230,8 @@ proptest! {
         // Cases are expensive, few cases is enough.
         // We will test a few more comprehensive tests more times, and the rest even fewer.
         // when trying to stress-test, increase (to 200 or more), and disable result cache.
-        cases: 10,
-        result_cache: prop::test_runner::basic_result_cache,
+        cases: 200,
```

Comment:
should the default be commented this way? @igor-aptos

Reply:
Oh, this is for extensive testing, too slow for CI. Let's revert these lines

Reply:
Done.